resources/worldconfig.ini

```diff
@@ -64,3 +64,15 @@ allow_nameplate_off=0
 
 # Turn logging of IP addresses for anti-cheat reporting on (1) or off(0)
 log_ip_addresses_for_anti_cheat=1
+
+help_0_summary=Got an issue?
+help_1_summary=Stuck loading?
+help_2_summary=Missing features?
+help_3_summary=Get smashed?
+help_4_summary=Want to contribute?
+
+help_0_description=Go to the DarkflameServer repository on Github to view issues and discussions about the server emulator!<br/><br/><a href="https://github.com/DarkflameUniverse/DarkflameServer"><font color="#0000EE">Click Here to go there!</font></a><br/><br/>
+help_1_description=Try switching networks or using your phone's hotspot to resolve the issue<br/><br/>
+help_2_description=Yep, while the server emulator is mostly complete, there are still some unimplemted parts that are being worked on. Such as property behaviors and Dragonmaw Chasm to name a couple.<br/><br/>
```

Comment:
unimplemented

Also, would change it to:

While DarkflameServer is a mostly complete emulator, there are still some features that aren't implemented. You can track these on the GitHub issues page.